src/FakeItEasy/Creation/DummyValueResolver.cs

```diff
@@ -69,7 +69,7 @@ public CreationResult TryResolveDummyValue(Type typeOfDummy, LoopDetectingResolu
             Type typeOfDummy,
             LoopDetectingResolutionContext resolutionContext)
         {
-            CreationResult creationResult = CreationResult.Untried;
+            CreationResult creationResult = null!;

```

Comment:
Seems like a minor and maybe silly change, but simplifies the logic in a future commit...